pages/lobby_page.py

```diff
@@ -244,3 +245,28 @@ def delete_ico(self):
     def click_delete_ico(self):
         for delete in self.delete_ico():
             delete.click()
+
+    def invite_team_form(self):
+        self.context.wait.until(lambda driver: driver.find_element_by_xpath('//a[text()="Invite your team"]'))
+        self.context.driver.find_element_by_xpath('//a[text()="Invite your team"]').click()
+        self.context.wait.until(EC.visibility_of_element_located((By.ID, 'invite-users-frame')))
+        self.context.driver.switch_to_frame(self.context.driver.find_element_by_id('invite-users-frame'))
+
+    def invite_team_email_input(self):
+        for add_email_try in range(0,3):
+            email = 'test' + str(randint(0, 999)) + '@send22u.info'
+            self.context.driver.find_element_by_id('email_input').send_keys(email, Keys.ENTER)
+            self.context.wait.until(EC.visibility_of_element_located((By.XPATH, '//td[text()="' + email + '"]')))
+
+    def delete_email_from_list(self):
+        delete_buttons = self.context.driver.find_elements_by_xpath('//a[text()="Remove"]')
+        choosen_but = choice(delete_buttons)
+        choosen_but.click()
```

Comment:
If I understand you right, at this method you want to check deletion of e-mail, this good idea, but I don't see any check, how do you know that randomly chosen email was deleted?

Reply:
I didnt make it with check if deleted, but added it. It could be better then without it. Thnx